### DISCUSSION TITLE
Access devices over the internet after encryption

### DISCUSSION THREAD

**Discussion by mvyas80** [rustdesk/rustdesk#7331] [General]
Enabled encryption with -K flag and have been using keys from .pub file but have had issues accessing devices over the internet. Devices on the same lan works fine over the encrypted connection but when tried to access devices over the internet. It is stuck at Connected, waiting for image attached is the screenshot.
![T1](https://github.com/rustdesk/rustdesk/assets/133155749/bf832220-9c00-4456-8768-759c11709837)

**Comment by xlionjuan** [comment]
Try both 1.2.3 and 1.2.4 to check it is same behaviour or not.

**Reply by mvyas80** [reply]
Hi I am trying it on 1.2.3 and cannot find 1.2.4.
Do you have a link or something where I can download it from?

**Reply by xlionjuan** [reply]
Find nightly

**Reply by mvyas80** [reply]
Thank you, yeah I tried that but looks like with that it does not work at all.

**Reply by xlionjuan** [reply]
Try regenerate the key

**Reply by mvyas80** [reply]
Yeah tried that too actually

**Reply by xlionjuan** [reply]
#7340 
Try new nightly, it seems similar to your issue
